content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -178,9 +178,9 @@ but there still may be changes in the API before stabilization making upgrades n
 guaranteed to be non-breaking.
 
 {{< note >}}
-Although the Device Manager component of Kubernetes is a generally available feature,
-the _device plugin API_ is not stable. For information on the device plugin API and
-version compatibility, read [Device Plugin API versions](/docs/reference/node/device-plugin-api-versions/).
+Although the Device Manager component of Kubernetes was graduated in v1.26,
+the _device plugin API_ is not stable (See [Slack discussion on Device Plugin API graduation](https://kubernetes.slack.com/archives/C0BP8PW9G/p1667481109075849) for details). For information on the device plugin API andversion compatibility, 
```

Comment:
Maybe we don't need the link to the Slack discussions.

Reply:
> @carlory can we mention the issue [k8s/k8s #113587](https://github.com/kubernetes/kubernetes/pull/113587) here, for better understanding of readers i.e. we Device Plugin API didn't graduate alongside the DeviceManager.

how about this? @tengqm 

```
{{< note >}}
Although the Device Manager component of Kubernetes was graduated in v1.26,
the _device plugin API_ doesn't graduate alongside the DeviceManager component, so it is not stable now. 
For information on the device plugin API and version compatibility, read [Device Plugin API versions](/docs/reference/node/device-plugin-api-versions/).
{{< /note >}}
```

Reply:
@carlory I want to mention the issue link, not the Slack discussion link.
@tengqm can we add the issue link: https://github.com/kubernetes/kubernetes/pull/113587

Reply:
For me the original text is already good. Why might we change it? Device Manager _is_ generally available, whether or not we delete the associated feature gate.

Reply:
> For me the original text is already good. Why might we change it? Device Manager _is_ generally available, whether or not we delete the associated feature gate.

Right. I raised the question for clarification. Since we are sure the API is not supposed to be treated as stable (GA), the existing text is fine, and can be left as is. No revision is needed.

Reply:
I want the issue link so that users know its background of it. otherwise, the text is fine.
if you think it is not needed then I am ok with that.

Reply:
By convention, we link to issues to record known defects (with a fix planned); otherwise we usually don't link to issues.

Reply:
@carlory, please revert the above changes back to the original text.